extras/kms/repository.go

```diff
@@ -130,6 +130,51 @@ type vetForWriter interface {
 	vetForWrite(ctx context.Context, opType dbw.OpType) error
 }
 
+func updateKeyCollectionVersion(ctx context.Context, w dbw.Writer) error {
+	const (
+		op = "kms.updateKeyCollectionVersion"
+
+		sql = "update kms_collection_version set version = version + 1"
+	)
+	if isNil(w) {
+		return fmt.Errorf("%s: missing writer: %w", op, ErrInvalidParameter)
+	}
+
+	rowsUpdated, err := w.Exec(ctx, sql, nil)
+	if err != nil {
+		return fmt.Errorf("%s: %w", op, err)
+	}
+	if rowsUpdated != 1 {
+		return fmt.Errorf("%s: update %q rows and expected 1: %w", op, rowsUpdated, ErrInternal)
+	}
+	return nil
```

Comment:
Would it be useful to return the current key version post-update? I realize it can always be changed later, just not sure if it would be helpful elsewhere in the current code.

Reply:
At this point, it's not helpful.  it's not exported, so we can always change the return in the future if/when it's helpful to return the post-update version.